containers/nginx-storage/delete-old-files.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+DIRECTORIES="$(echo /srv/storage/blender/{result/,'source/',verifier-output/})"
+
+for directory in ${DIRECTORIES[@]}; do
+    if [ ! -d "$directory" ]; then
+        echo "[crond][error][disk cleanup]: Please check if the \"$directory\" directory exist."
+        exit 1
+    fi
+done
+
+free_space="$(df -h | grep '/srv/storage' | awk '{ print $4; }' | tr -d 'G')"
```

Comment:
does it remove other characters other than 'G' like 'M' (megabytes) ? maybe better would be to remove -h (from df), and then parse raw bytes?

Reply:
Done.